images/gcsfuse/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM ubuntu:22.04
+
+ARG VERSION=${VERSION:-1.0}
+
+RUN apt-get update && \
+    apt-get install -y lsb-release wget && \
+    GCSFUSE_REPO=gcsfuse-$(lsb_release -c -s) && \
+    echo "deb http://packages.cloud.google.com/apt $GCSFUSE_REPO main" | tee /etc/apt/sources.list.d/gcsfuse.list && \
```

```suggestion
    apt-get install -y gnupg lsb-release wget && \
    GCSFUSE_REPO=gcsfuse-$(lsb_release -c -s) && \
    echo "deb https://packages.cloud.google.com/apt $GCSFUSE_REPO main" | tee /etc/apt/sources.list.d/gcsfuse.list && \
```

I tried this locally and it wouldn't build without gnupg/similar installed

```E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation```